slack/client.go

```diff
@@ -108,14 +119,25 @@ func (c *Client) RunHistoricalBackfilling(ctx context.Context) error {
 
 // Export sends slack message to the additional service via callback
 func (c *Client) Export(cb func(m Message)) {
+	ticker := time.NewTicker(batchSendInterval)
 	for {
 		select {
+		case <-ticker.C:
+			log.Printf("sending batch of %d messages", len(c.batch))
+			c.mx.Lock()
+			for _, m := range c.batch {
+				cb(m)
+				messageOutCount.Inc()
+			}
+			maps.Clear(c.batch)
```

Comment:
Since go 1.21 there is a built-in clear implementation - https://pkg.go.dev/builtin#clear, so the code can be updated to:
```suggestion
			clear(c.batch)
```

And `/x/exp` import can be removed.

Reply:
fixed